build.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python2.7
+#!/usr/bin/env python
```

Comment:
- In general I think we should support Python 2.x (which we do). I don't think we used anything 2.7-specific so it'd be better to keep it general.

- python2.7 is a Linux distro-specific name.